mkdocs.yml

```diff
@@ -64,7 +64,7 @@ nav:
     - CHOS: 
         - 'Overview': 'epidemiology/covid_19_hospital_occupancy_simulator/overview.md'
         - 'Guide': 
-          - 'Simulation Flow': 'epidemiology/covid_19_hospital_occupancy_simulator/simulation_flow.md'
+          - 'How Simulations Work': 'epidemiology/covid_19_hospital_occupancy_simulator/simulation_flow.md'
```

Comment:
Can you replace `How Simulations Work` with `How do simulations work?`

As well, can you change the name of the file from `simulation_flow.md` to `how_do_simulations_work.md`